fontbakery-check-ttf.py

```diff
@@ -4204,6 +4325,40 @@ def fontbakery_check_ttf(config):
                "  --error\tPrint only the error messages "
                "(outputs to stderr).\n")
 
+##########################################################
+## Step 3: Regression related checks:
+# if family already exists on fonts.google.com
+##########################################################
+
+  # ------------------------------------------------------
+  if check_METADATA_Fontfamily_is_listed_in_GoogleFontDirectory(fb, family):
+    remote_fonts_zip = download_family_from_GoogleFontDirectory(family.name)
+    remote_fonts_to_check = fonts_from_zip(remote_fonts_zip)
+
+    remote_styles = {}
+    for target in remote_fonts_to_check:
+      remote_font = target.get_ttfont()
+      remote_family, remote_style = target.fullpath[:-4].split('-')
+      remote_styles[remote_style] = remote_font
+
+      # Only perform tests if local fonts have the same styles
+      if remote_style in local_styles:
+        check_regression_v_number_increased(
+          fb,
+          local_styles[style],
+          remote_styles[style],
+          f
+        )
+        check_regression_glyphs_structure(
+          fb,
+          local_styles[style],
+          remote_styles[style],
+          f
+        )
+        fb.output_report(target)
```

Comment:
there's a bad side-effect here... This will mix check-results reports for same target between local and remote sources. We need to figure out a way to distinguish these because these reports will end up eighter in json/md report files in the local filesystem or in the burndown chart json data file.

Maybe it is enough to prepend "GFONTS_" to the target name here. Or maybe we could refactor the output_report method to receive a file_source parameter with a default "local" value, and do something here similar to:

```python
fb.output_report(target, file_source="gfonts")
```